src/components/App/App.tsx

```diff
@@ -25,26 +29,33 @@ export const AppComponent: React.StatelessComponent<{}> = () => {
 
     return (
         <ThemeProvider theme={muiTheme}>
-            <APIContext.Provider value={apiState}>
-                <SkeletonTheme
-                    color={skeletonColor}
-                    highlightColor={skeletonHighlightColor}
-                >
-                    <CssBaseline />
-                    <Helmet>
-                        <title>Flyte Console</title>
-                        <meta name="viewport" content="width=device-width" />
-                    </Helmet>
-                    <Router history={history}>
-                        <ErrorBoundary fixed={true}>
-                            <NavBarRouter />
-                            <ApplicationRouter />
-                            <LoginExpiredHandler />
-                        </ErrorBoundary>
-                    </Router>
-                    <SystemStatusBanner />
-                </SkeletonTheme>
-            </APIContext.Provider>
+            <QueryClientProvider client={queryClient}>
+                <APIContext.Provider value={apiState}>
+                    <QueryAuthorizationObserver />
+                    <SkeletonTheme
+                        color={skeletonColor}
+                        highlightColor={skeletonHighlightColor}
+                    >
+                        <CssBaseline />
+                        <Helmet>
+                            <title>Flyte Console</title>
+                            <meta
+                                name="viewport"
+                                content="width=device-width"
+                            />
+                        </Helmet>
+                        <Router history={history}>
+                            <ErrorBoundary fixed={true}>
+                                <NavBarRouter />
+                                <ApplicationRouter />
+                                <LoginExpiredHandler />
+                            </ErrorBoundary>
+                        </Router>
+                        <SystemStatusBanner />
+                    </SkeletonTheme>
+                </APIContext.Provider>
+                <ReactQueryDevtools initialIsOpen={false} />
```

Comment:
Should this only be enabled in `dev`?

Reply:
According to the docs, the import is a no-op in production, so this is safe to leave in.

Reply:
I will try building prod mode locally to verify, and if that's not the case, I'll add logic in the next PR to conditionally render it.